scripts/package.py

```diff
@@ -146,6 +152,12 @@ def main(*additional_dist_formats: Format, verbosity: int = 0, serve: bool = Fal
     action='append',
     help='Package Pex in additional formats.'
   )
+  parser.add_argument(
+    '--local',
+    default=False,
+    action='store_true',
+    help='Build Pex PEX with just a single local interpreter.'
```

Comment:
Is it clear in this context which one you'll get?

Reply:
Yes, unconstrained `pex ...` uses the interpreter it was invoked with which is sys.executable above. The other factor is Pex is multipython and not native, so ~any interpreter will do.

To be clear - this is working around the fact --interpreter-constraint has been broken since inception for Pex builtime since its always been used to pick the list of interpreters used instead of sanity check them. It is fully coherent at runtime in contrast.

This just stops bleeding on a machine like mine where I have ~15 pythons installed which brings everything (including the cursor) to a halt for many minutes.